backup.html

```diff
@@ -0,0 +1,271 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Twitter</title>
+</head>
+<body>
+    <div class="container">
+        <div class="left">
+            <div class="left-container">
+                <div class="menu">
+                    <div class="menu-header">
+                        <div class="logo-box">
```

Comment:
Lots of divs! Hopefully this morning session on semantic elements will help here next time :)